README.md

```diff
@@ -41,7 +41,7 @@ val text = env.socketTextStream(host, port, '\n')
 val windowCounts = text.flatMap { w => w.split("\\s") }
   .map { w => WordWithCount(w, 1) }
   .keyBy("word")
-  .window(TumblingProcessingTimeWindow.of(Time.seconds(5)))
```

Comment:
remove scala example?

Reply:
I will leave that to [FLINK-36352](https://issues.apache.org/jira/browse/FLINK-36352)